lib/src/test_flow_control_translations.dart

```diff
@@ -125,6 +141,11 @@ class TestFlowControlTranslations {
     value: 'Fail',
   );
 
+  static const atf_flow_title_include_test = TranslationEntry(
+    key: 'atf_flow_title_include_test',
+    value: 'Include Test',
```

Comment:
Question: "Include Test" or "Include Sub-Test"?

I don't have a strong opinion either way and am ok with whichever you go with.